aws/cloudformation-templates/base/_template.yaml

```diff
@@ -57,6 +57,11 @@ Parameters:
     Description: Segment source write key (optional).
     NoEcho: true
   
+  XDNDeployToken:
+    Type: String
+    Description: Moovweb XDN token for deploying (optional).
+    NoEcho: true
+  
```

Comment:
@james-jory did I define `XDNDeployToken` correctly everywhere through `.yaml` files?

Reply:
Yes, I _think_ you hit all the right places but you should make sure to test the deployment with and without XDN enabled to make sure that both paths deploy _and_ operate correctly.